### PR TITLE
Update safe-app dependencies

### PR DESCRIPTION
**Pull request by bh2smith**
Only had to change `safe.network` to `safe.chainId` and make related changes.

### CODE REVIEW

File: src/__tests__/tokenList.test.ts [hunk 1]
```diff
@@ -2,12 +2,12 @@ import { expect } from "chai";
 
 import { fetchTokenList } from "../hooks/token";
 
-const configuredNetworks = ["MAINNET", "RINKEBY", "XDAI"];
+const configuredNetworks = [1, 4, 100];
```
Comment (schmanu) [inline src/__tests__/tokenList.test.ts:5]: How did you find out those magic numbers? Should we probably have some constants for these?

**Review by schmanu — APPROVED** [review]
Only have one minor suggestion. 
Otherwise it looks fine to me :) Always nice to keep those dependencies up to date!

File: src/__tests__/tokenList.test.ts (hunk 1 above)
Comment (bh2smith) [reply src/__tests__/tokenList.test.ts:5]: These are standard values for those networks. In the token list file I pus a constant call networkMap. We could import this and use it here.

Comment (bh2smith) [reply src/__tests__/tokenList.test.ts:5]: Also might be able to put this in a project config file. I'll try it out later.

**Pull request merged**